functions/apex-restart.py

```diff
@@ -0,0 +1,33 @@
+import boto3
+import os
+import json
+
+def lambda_handler(event, context):
+    client = boto3.client('ecs')
+    cluster=os.environ.get('CLUSTER')
+    service=os.environ.get('SERVICE')
+    security_group=os.environ.get('SECURITY_GROUP')
+    subnet_one=os.environ.get('SUBNET_ONE')
+    subnet_two=os.environ.get('SUBNET_TWO')
+
+    print("replacing apex service...")
+    response = client.update_service(
+        cluster=cluster,
+        service=service,
+        desiredCount=2,
+        taskDefinition=service,
+        networkConfiguration={
```

Comment:
Ideally, we'd just have a `forceNewDeployment=True`

Is it possible to _just_ pass `forceNewDeployment=True`, or if not, how about you make an initial describe call to fetch the existing config based on the cluster/service name, then use that to populate the `update_service` and avoid that extra config.

Reply:
Removed the non required attributes